packages/dependency-royalty-management/contracts/RoyaltyDistributer.sol

```diff
@@ -14,16 +14,13 @@ contract RoyaltyDistributer is IERC2981Upgradeable {
 
```

Comment:
Should this contract be `abstract` like the MultiRoyaltyDistributer.sol?

Reply:
No undefined methods

Reply:
What do you mean by that?

Reply:
An abstract contract has methods that are not defined i.e they have interface of a method but not the complete method.

Reply:
@rishabh0x00 

That is true for interfaces, abstract contracts can have implementation.
Take a look at your `MultiRoyaltyDistributor.sol`, it's an abstract contract.

Reply:
It is an abstract contract because it has methods which are define in the contract which inherits it. i.e Asset

Reply:
Alright, let's leave it as is, since none of the functions are overriden it may make sense to leave it as regular contract.
Thanks @rishabh0x00